src/main/java/com/easypost/model/ShipmentMessage.java

```diff
@@ -3,26 +3,29 @@
 public class ShipmentMessage {
   String carrier;
   String type;
-  String message;
+  Object message;
```

Comment:
Changing this to an object would not be backwards compatible and thus this would not be a minor version bump. This should at least be a mid version bump but I am more in favor of a major version bump.

Reply:
I think a mid version bump would be applicable here, since historically we've had major version bumps if there were any changes in the dependent libraries.